libr/bin/bfile.c

```diff
@@ -852,20 +852,19 @@ R_API bool r_bin_file_close(RBin *bin, int bd) {
 R_API int r_bin_file_hash(RBin *bin, ut64 limit, const char *file) {
```

Comment:
the function signature returns int, shouldnt this be bool instead?

Reply:
yes good point.